src/vs/platform/dialogs/common/dialogs.ts

```diff
@@ -240,7 +240,7 @@ export interface IOpenDialogOptions {
 	/**
 	 * A human-readable string for the open button.
```

Comment:
Should this also apply to `saveLabel` above?

Reply:
Yes, but I was trying to keep this change scoped so I could use the same change for the release branch. I'll just make the change and remove it in the release branch.